test/commerce.spec.ts

```diff
@@ -146,7 +146,7 @@ describe('commerce', () => {
           expect(
             amount,
             'The expected match should not include a currency symbol'
-          ).match(/[0-9.]/);
+          ).toMatch(/[0-9.]/);
```

Comment:
Not sure whether we should fix it here or in a separate PR, but this should probably look like this:

```suggestion
          ).toMatch(/^[0-9\.]+$/);
```

Reply:
Fixed in #865